spec/lib/pubmed/fetcher_spec.rb

```diff
@@ -95,8 +96,11 @@
       end
     end
 
-    context 'WOS enabled' do
-      before { allow(Settings.WOS).to receive(:enabled).and_return(true) }
+    context 'Pubmed and WOS both enabled' do
+      before do
+        allow(Settings.WOS).to receive(:enabled).and_return(true)
+        allow(Settings.PUBMED).to receive(:lookup_enabled).and_return(true)
+      end
 
       it 'searches only WOS' do
```

Comment:
if they are both enabled, shouldn't it search both?

Reply:
its a bit confusing, but not in this case

The reason is that

1. You can search for publications by PMID in both WoS and Pubmed Central.
2. WoS is currently the preferred service when you search by PMID (something that is actually changed by one of the draft PRs, but would require verification by the Profiles team)
3. This test has a result coming from WoS for the PMID being searched.

So in the case when both are enabled, you'll get a result from WoS, then stop and never hit PMID.

Really this scenairo would have another test that leaves them both enabled, but simulate returning nothing from WoS.  Then we *would* expect the Pubmed client to run.

Reply:
lemme take a stab at writing such a test

Reply:
and will clarify what is happening in the test too with a better test name

Reply:
actually the very next test does test this scenario of no results found at WoS and verifies it hits Pubmed.  let me word for better clarity though